solutions/README.md

```diff
@@ -1 +1,111 @@
-# Solutions
+# **Solutions Folder Overview**
+
+The **Solutions** folder contains implementations of coding challenges completed by our group. Each solution file directly addresses a specific challenge, while corresponding test files are maintained in the `tests` folder.
+
+---
+
+## **Folder Structure**
+
+    └── solutions
+        ├── README.md
+        ├── __init__.py
+        └── tests
+            ├── README.md
+            └── __init__.py
+
+
+---
+
+## **Solutions Summary**
+
+| **Solution File**       | **Description**                                                | **Author**        |
+|--------------------------|--------------------------------------------------------------|-------------------|
+| `add_numbers.py`         | Adds two numbers and returns the result.                     | [Author Name]     |
+| `calculate_factorial.py` | Computes the factorial of a given number using recursion.    | [Author Name]     |
+| `string_reversal.py`     | Reverses a given string and returns the reversed output.     | [Author Name]     |
+
+---
+
+# **How to Update**
```

Comment:
this should be a level 2 header and should have a blank space above and below it.